### ISSUE TITLE
Incorrect error reporting, part 2

### ISSUE THREAD

**Issue by julianjensen** [cherow/cherow#49]
Your fix to my previous issue was impressively prompt and I think it might be working but I can't be sure because now I'm getting a similar error elsewhere. This code is (almost) unmodified from my original source but it's still fairly small. I will paraphrase the exception it throws as I can't easily copy anything from my work machine to, well, anywhere. This fails in the latest version from today (0.11.5):

```js
require( 'cherow' ).parseModule( `const
    sequential = ( ...fns ) => ( ...args ) => fns.map( fn => fn( ...args ) ), 
    pchain = ( ...fns ) => fns.map( f => isPromise( f ) ? f : pdefer( f ) ).reduce( ( prev, cur ) => prev.then( cur ) );`, { next: true } );
```

The error roughly is in your `dist/cherow.js:444` (probably not helpful) and says:

Error: Line 3: 'fns' has already been declared.

Sorry I can't attach a complete error message from the thrpw, but my work environment is very tightly controlled, nothing goes in or out, everything is monitored, &c.

I guess my coding style is susceptible to whatever this error is.

Julian

**Comment by KFlash** [comment]
Closing as it should have been fixed by now.

**Comment by KFlash** [comment]
@julianjensen Try download latest NPM version of Cherow. Just pushed it. Your latest issue was fixed 3.56AM in the morning my time, so I was not fully awake when I fixed that. But I improved the fix now.
